website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step05-controller-iam.sh

```diff
@@ -2,6 +2,7 @@ eksctl create iamserviceaccount \
   --cluster "${CLUSTER_NAME}" --name karpenter --namespace karpenter \
   --role-name "${CLUSTER_NAME}-karpenter" \
   --attach-policy-arn "arn:aws:iam::${AWS_ACCOUNT_ID}:policy/KarpenterControllerPolicy-${CLUSTER_NAME}" \
+  --attach-policy-arn "arn:aws:iam::${AWS_ACCOUNT_ID}:policy/KarpenterEventPolicy-${CLUSTER_NAME}" \
```

Comment:
Were you planning on doing the upgrade notes in a different PR? 

We'll need to make clear that to upgrade Karpenter they'll need to attach the new policy to the controller role since eksctl create iamserviceaccount doesn't add attachments if the role is already created.

```
> aws iam attach-role-policy --role-name "${CLUSTER_NAME}-karpenter" --policy-arn "arn:aws:iam::${AWS_ACCOUNT_ID}:policy/KarpenterEventPolicy-${CLUSTER_NAME}"
```

The upgrade guide or maybe a dedicated docs section on Spot should include migration from NTH to Karpenter native interruption handling since that will be useful for new installs of Karpenter and the upgrade case.

Reply:
Yeah, I was planning to follow-up this PR with a docs PR as a fast-follow. I figured there would be a lot of discussion on wording for migration guide and the docs PR would be relatively large.